src/bdfs/bdfs.conf

```diff
@@ -0,0 +1,6 @@
+{
+  "kademlia" : [ "http://localhost:7800" ],
```

Comment:
Talk to victor for public kademlia address in US. We should include that when we create installation package for client. (Better to have a build flag to specify something else for test builds, especially for Shanghai)